src/cmap/connection.ts

```diff
@@ -237,6 +237,8 @@ export class Connection extends TypedEventEmitter<ConnectionEvents> {
       .on('error', this.onError.bind(this));
     this.socket.on('close', this.onClose.bind(this));
     this.socket.on('timeout', this.onTimeout.bind(this));
+
+    this.messageStream.pause();
```

Comment:
Streams start paused, so I'm happy to remove this.  This just made it a bit more explicit - without this and knowing that streams start paused, it wasn't immediately obvious how the new tests pass for the first message from the message stream.

Reply:
> Streams start paused, so I'm happy to remove this.

(This probably isn't particularly relevant to this particular PR, but just as a fun fact, stream's don't act as if `.pause()` or `.resume()` had been called on them initially; instead, they are in a secret third state (in particular, `.on('data', ...)` implicitly starts reading, which it doesn't if it's a `.pause()`d stream). A topic that's been brought up from time to time in Node.js core is actually that there's no way to "reset" a stream into its initial state once either of these methods have been called 🙂)

Reply:
That's actually included in the documentation now (maybe it has been for a while?  I found it earlier today): https://nodejs.org/api/stream.html#implementing-a-transform-stream:~:text=data%27%20event%20handler.-,Three%20states,-%23

But for us, it just means that we won't miss data events on the first message.  Subsequent messages can miss data events, because `onData` removes its `data` listener but doesn't pause the message stream in its cleanup logic.

Reply:
I'm fine with calling pause out the gate, I don't think we have use for that "third" state, but very interesting to know!